src/main/java/amazin/security/WebSecurityConfig.java

```diff
@@ -30,17 +29,11 @@ public AuthenticationManager customAuthenticationManager() throws Exception {
 
     @Override
     protected void configure(HttpSecurity http) throws Exception {
-        http
-            .authorizeRequests()
-                .antMatchers("/webjars/**", "/register", "/admin/register", "/forgot-password").permitAll()
-                .anyRequest().authenticated()
-                .and()
-            .formLogin()
-                .loginPage("/login")
-                .permitAll()
-                .and()
-            .logout()
-                .permitAll();
+        http.authorizeRequests()
+                .antMatchers("/webjars/**", "/register", "/admin/register", "/forgot-password", "/search", "/",
+                        "/js/**", "/css/**", "/images/**")
+                .permitAll().anyRequest().authenticated().and().formLogin().loginPage("/login").permitAll().and()
+                .logout().permitAll();
```

Comment:
again for this PR, the only thing that was changed here was the string within .antMatchers() but due to all these indentation changes it looks like the entire method was changed, makes it more difficult to figure out what was changed for the PR reviewer...

Reply:
adding this java linter dependency to the Amazin Wiki:
![java-redhat-linter](https://user-images.githubusercontent.com/5737382/55300810-68b97680-5407-11e9-8ed0-03dc9d53aa12.png)